README.md

```diff
@@ -139,31 +142,33 @@ As the example gave `save_path` argument to the last block of `ETL_config`, data
-|           | decontamination | (WIP) Remove contaminated data including benchmark.                                               |                 |
-|           | deduplication   | Remove duplicated data, targeting not only identical matches but also similar data.               | 4               |
-|           | pii             | PII stands for Personally Identifiable Information. Removing sensitive information from data.     | 2               |
-|           | quality         | Improving the data quality, in the perspective of accuracy, consistency, and reliability of data. | 1               |
-|           | toxicity        | (WIP) Removing harmful, offensive, or inappropriate content within the data.                      |                 |
-| Load      | data_load       | Saving the processed data to a preferred source like data lake, database, etc.                    | 4               |
-| Utils     | utils           | Essential tools for data processing, including sampling, logging, statistics, etc.                | 3               |
+Currently, about 50 functions are registered as the ETL process, which means they are eagerly awaiting your use!
+| Type      | Package         | description                                                                                       |
+|-----------|-----------------|---------------------------------------------------------------------------------------------------|
+| Extract   | data_ingestion  | Loading data from any source to the preferred format                                              |
+| Transform | bias            | (WIP) Reduce skewed or prejudiced data, particularly data that reinforce stereotypes.             |
+|           | cleaning        | Remove irrelevant, redundant, or noisy information, such as stop words or special characters.     |
+|           | decontamination | (WIP) Remove contaminated data including benchmark.                                               |
+|           | deduplication   | Remove duplicated data, targeting not only identical matches but also similar data.               |
+|           | pii             | PII stands for Personally Identifiable Information. Removing sensitive information from data.     |
+|           | quality         | Improving the data quality, in the perspective of accuracy, consistency, and reliability of data. |
+|           | toxicity        | (WIP) Removing harmful, offensive, or inappropriate content within the data.                      |
+| Load      | data_load       | Saving the processed data to a preferred source like data lake, database, etc.                    |
+| Utils     | utils           | Essential tools for data processing, including sampling, logging, statistics, etc.                |
 <br>
 
-
-## 🌌 AWS Support
+## 🌌 Dataverse supports AWS
 Dataverse supports AWS! Step by step guide to setting up is [here](https://data-verse.gitbook.io/docs/lets-start/aws-s3-support).
```

Comment:
AWS just refers Amazon Web Services. "Dataverse supports AWS!" should be written in more detail about what features AWS supports.